### DISCUSSION TITLE
Separate losses for each node for node classification

### DISCUSSION THREAD

**Discussion by gregoritoo** [pyg-team/pytorch_geometric#5694] [Q&A]
Hi!
I would like to backpropagate different losses for each node to learn separate edge attention masks. My edge attention mask is a matrix of shape (number_of_nodes, number_of_edges) and i would like to update each row separately. Is there a simpler way to do it ?

**Comment by rusty1s** [comment]
It is not totally clear to me what you mean. If you want to learn node-specific parameters, you could think about learning an embedding matrix that is used to define edge attention scores.
